mne/decoding/gsoc.py

```diff
@@ -0,0 +1,195 @@
+# Authors: Jean-Remi King <jeanremi.king@gmail.com>
+#
+# License: BSD (3-clause)
+
+import numpy as np
+from .mixin import TransformerMixin
+from ..epochs import _BaseEpochs
+
+
+class EpochsTransformerMixin(TransformerMixin):
+    """Mixin class for reshaping data to Epoch's standard shape
+
+    This class is meant to be inherited by transformers meant to be
+    used in scikit-learn pipeline. Useful utility for making data
+    compatible with mne internal functions.
+
+    Parameters
+    ----------
+    n_chan : int | None
+
```

Comment:
* remove empty line
* add `(default : None)`,
* add what is used for (reshaping) 
* add explanation of what happens if `None`